### DISCUSSION TITLE
Huawei EMMA A02 + 10K-MAP0 + LUNA S1 - Greyed out things

### DISCUSSION THREAD

**Discussion by cybermailer84** [wlcrs/huawei_solar#862] [Setting up the integration]
Hi, used the old integration and updated to the new one, by replacing the files and folder from huawei_solar integration.

Then i installed the integration - configuration for modbus tcp (slave auto) and i got EMMA, Batteries, Batterie-1 and MAP0.

Now i saw, that some important things are "not usable / deactivated" is this normal, how can i use it.

Specially i want to test AC charging. I actived and tested this option already with fusionsolar (web) - this works. How to do with huawei_solar integration?

thx

![Screenshot 2024-12-05 195958](https://github.com/user-attachments/assets/0a202780-ab6d-48fb-ad45-45a94ee62e14)
![Screenshot 2024-12-05 200129](https://github.com/user-attachments/assets/10f09bd1-01e3-4a5a-a5ef-6e334d9fc78b)

**Comment by mithri** [comment]
Is AC charging even available with EMMA? I cannot find any setting in the FusionSolar app or on the portal.

**Reply by cybermailer84** [reply]
My question ist obsolete - it does work. 

To your question yes it is. Login into webportal of Fusion solar. And then go to settings of your battery.
